modules/ocf_kvm/facts.d/vms

```diff
@@ -11,7 +11,9 @@ def run_command(command_args):
     return list(filter(None, output))
 
 
-if __name__ == '__main__' and os.path.isfile('/usr/bin/virsh'):
+if (__name__ == '__main__' and
+    os.path.isfile('/usr/bin/virsh') and
+        subprocess.call(('/bin/systemctl', 'is-active', '--quiet', 'libvirtd')) == 0):
```

Comment:
Indentation here seems kinda strange, I'd probably do something like this, with the added benefit that you can add comments about why these checks actually exist:

```
if __name__ == '__main__':
    # Only check for VMs if virsh is actually available
    if not os.path.isfile('/usr/bin/virsh'):
        return

    # Don't check for VMs if libvirtd isn't running
    if subprocess.call(('/bin/systemctl', 'is-active', '--quiet', 'libvirtd')) != 0:
        return

    [...]
```

Reply:
Not sure what's up with the indentation to be honest, it was automatically done by flake8 iirc. I like you approach better.